cli/defenseclaw/commands/windows_native_uninstall.py

```
@@ -0,0 +1,760 @@
+        or payload.get("unsigned") is not False
+        or not isinstance(payload.get("authenticode"), dict)
+        or not isinstance(payload.get("toolchain"), dict)
+        or not isinstance(payload.get("files"), dict)
+    ):
+        raise NativeWindowsUninstallRefusal(
+            "Native payload manifest does not match signed installer-state custody."
+        )
+
+
+@contextlib.contextmanager
+def _open_locked_setup(path: str) -> BinaryIO:
```

Ruff (0.16.0)

[warning] Too many statements (57 > 50)

(PLR0915)